pallets/ethereum-transactions/src/lib.rs

```diff
@@ -447,9 +425,9 @@ impl<T: Config> Pallet<T> {
 
     fn get_contract_address(transaction_type: &EthTransactionType) -> Option<H160> {
         return match transaction_type {
```

Comment:
Should we simplify this code and always return the bridge contract instead? We can also eliminate the function altogether if we want and just access the storage item from avn pallet.

Reply:
I am not too sure - the enum 'EthTransactionType' has many different values, including 'Invalid' in which an empty address should be returned.